ddl/syncer.go

```diff
@@ -380,3 +398,103 @@ func (s *schemaVersionSyncer) OwnerCheckAllVersions(ctx context.Context, latestV
 		time.Sleep(checkVersInterval)
 	}
 }
+
+const (
+	opDefaultRetryCnt = 10
+	failedGetTTLLimit = 20
+	neededCleanTTL    = -60
+	opDefaultTimeout  = 3 * time.Second
+	opRetryInterval   = 500 * time.Millisecond
+)
+
+func (s *schemaVersionSyncer) StartCleanWork() {
+	for {
+		time.Sleep(time.Second * 1)
+		if !s.ownerChecker.IsOwner() {
+			continue
+		}
+
+		select {
+		case <-s.notifyCleanExpiredPathsCh:
```

Comment:
We can remove all the pending msg in the s.notifyCleanExpiredPathsCh, to avoid running too frequently

Reply:
The `notifyCleanExpiredPathsCh` length is 1. And maybe we do this case take a long time, maybe find another key need to be removed.

Reply:
got it.